shared-libs/cht-datasource/src/index.ts

```diff
@@ -85,6 +85,20 @@ export const getDatasource = (ctx: DataContext) => {
          * @throws Error if no UUID is provided
          */
         getByUuidWithLineage: (uuid: string) => ctx.bind(Person.v1.getWithLineage)(Qualifier.byUuid(uuid)),
+
+        /**
+         * Returns an array of people for the provided page specifications.
+         * @param personType the type of people to return
+         * @param limit the maximum number of people to return. Default is 100.
+         * @param skip the number of people to skip. Default is 0.
+         * @returns an array of people for the provided page specifications
+         * @throws Error if no type is provided or if the type is not for a person
+         * @throws Error if the provided limit is `<= 0`
+         * @throws Error if the provided skip is `< 0`
+         */
+        getPage: (personType: string, limit = 100, skip = 0) => ctx.bind(Person.v1.getPage)(
```

Comment:
Another after-thought I had here was that we should probably name this function _getPageByType_. 

My vision is that in the future the `Person.v1.getPage` function could be expanded to accept different Qualifiers without changing the function name (e.g. you could get a page of persons by birthDate or something).  We should be able to keep things from getting too confusing there because of the structure provided by the qualifier objects/types.  

However, for these functions we are directly accepting the parameters (and don't want to do anything sketchy like `string | Date`).  So, we will end up with one function per type of parameter combination.  With that in mind, we should name things accordingly....